### DISCUSSION TITLE
Heavy model issue with exported model

### DISCUSSION THREAD

**Discussion by bakerwy** [fumitoh/modelx#114] [Q&A] [answered]
Hi, 

I have built very heavy. Also, I am trying to figure out issues of modelx and compare the results with other actuarial programs.

First of all, I built model using vectorized approach for the purpose of speed up.

Thankfully, It works well when I use the model with Spyder or ipython without export.
But with exported model, it works (e.g. calling variables and tables) but when I try to calculate heavy variables, python suddenly switched off without any message. I guess it is sort of stack overflow. (Not sure though)

But, what I don't understand is the model works well. so only stack overflow doesn't fully explain this...
Is there any solution to solve this problem or any advises?

**Comment by fumitoh** [comment, marked as answer]
What version of Python are you using? If the recursions in your model is too deep, then Python older than 3.11 may crash.
3.11 and 3.12 and future versions of Python should work fine.

**Reply by bakerwy** [reply]
Yes it is working above 3.11. Thanks!
